flink-core/src/test/java/org/apache/flink/api/common/operators/DualInputSemanticPropertiesTest.java

```diff
@@ -165,12 +164,12 @@ public void testGetSourceField() {
         sp.addForwardedField(0, 1, 2);
         sp.addForwardedField(0, 1, 3);
 
-        assertEquals(0, sp.getForwardingSourceField(0, 0));
-        assertEquals(0, sp.getForwardingSourceField(0, 4));
-        assertEquals(1, sp.getForwardingSourceField(0, 1));
-        assertEquals(1, sp.getForwardingSourceField(0, 2));
-        assertEquals(1, sp.getForwardingSourceField(0, 3));
-        assertTrue(sp.getForwardingSourceField(0, 5) < 0);
+        assertThat(sp.getForwardingSourceField(0, 0)).isZero();
+        assertThat(sp.getForwardingSourceField(0, 4)).isZero();
+        assertThat(sp.getForwardingSourceField(0, 1)).isEqualTo(1);
+        assertThat(sp.getForwardingSourceField(0, 2)).isEqualTo(1);
+        assertThat(sp.getForwardingSourceField(0, 3)).isEqualTo(1);
+        assertThat(sp.getForwardingSourceField(0, 5) < 0).isTrue();
```

Comment:
```suggestion
        ```suggestion
        assertThat(sp.getForwardingSourceField(0, 1)).isOne();
        assertThat(sp.getForwardingSourceField(0, 2)).isOne();
        assertThat(sp.getForwardingSourceField(0, 3)).isOne();
        assertThat(sp. getForwardingSourceField(0, 5)).isNegative();
```
```